CMakeLists.txt

```diff
@@ -61,20 +62,18 @@ else()
     set( iodaconv_pygrib_ENABLED False)
 endif()
 
-# Optional:  eccodes for Python (and C libs)
-option(USE_ECCODES "Use Python eccodes package" OFF)
-if ( USE_ECCODES )
-    find_package( eccodes QUIET )
-    if ( eccodes_FOUND )
-        message(STATUS "Adding python tests that import eccodes")
+find_package( eccodes QUIET)
+
+if ( eccodes_FOUND )
+        message(STATUS "eccodes found")
         set( iodaconv_eccodes_ENABLED True)
-    else()
-        message(STATUS "Not-found: eccoodes - eccodes disabled")
-        set( iodaconv_eccodes_ENABLED False)
-    endif()
-else()
-    message(STATUS "Skipping python tests that import eccodes")
-    set( iodaconv_eccodes_ENABLED False)
+endif()
+
+find_package( cartopy QUIET )
+
+if ( cartopy_FOUND )
+        message(STATUS "cartopy found")
+        set( iodaconv_cartopy_ENABLED True)
```

Comment:
```suggestion
        set( iodaconv_cartopy_ENABLED True)
```
```suggestion
        set(iodaconv_cartopy_ENABLED True)
```

Reply:
I should probably delete these enviromental variables just in case someone tries to use them later. Thanks for the catch!